packages/aws-cdk/lib/contextplugins.ts

```diff
@@ -64,5 +64,6 @@ export async function provideContextValues(missingValues: { [key: string]: Missi
 
     const value = await provider.getValue(query.scope, query.args);
     projectConfig.set(['context', key], value);
+    debug(`Setting "${key}" context to ${value}`);
```

Comment:
Might want to `JSON.stringify(value)`, in case it's one of those things that `.toString()`'s to `[Object object]`.

Reply:
👍